pages/linux/renice.md

```diff
@@ -11,11 +11,11 @@
 
 - Increase the priority of a running process:
```

Comment:
Try something like `- Increase the priority of a running process (lower value is higher priority:` so that this doesn't happen again with someone else.